### PR TITLE
Issue #ED-2511 fix: Disappearance Issue of "Browse Other Categories" text in the dark mode is fixed

### PR DESCRIPTION
**Pull request by mithun30052001**
# SunbirdEd - Portal
---

### Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
---

#### Example
- [x] Applicable
- [ ] Not applicable

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts [hunk 1]
```diff
@@ -130,8 +134,21 @@ export class ExplorePageComponent implements OnInit, OnDestroy, AfterViewInit {
...
+        this.themeSubscription = this.theme$.subscribe((theme) => {
+        });
+        
+        const localDataThemeAttribute = localStorage.getItem('data-mode');
+        if (localDataThemeAttribute) {
+          this.setTheme(localDataThemeAttribute);
+        }
     }
 
+    setTheme(value: string) {
```
Comment (princegupta1131) [inline src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts:147]: @mithun30052001  please add the comment description

File: src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts [hunk 2]
```diff
@@ -130,8 +134,21 @@ export class ExplorePageComponent implements OnInit, OnDestroy, AfterViewInit {
                 this.userType = result;
             }
         });
+
+        this.themeSubscription = this.theme$.subscribe((theme) => {
```
Comment (princegupta1131) [inline src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts:138]: what is the use of this line of code I am not seeing and use 
   this.themeSubscription = this.theme$.subscribe((theme) => {
        });

File: src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts (hunk 1 above)
Comment (mithun30052001) [reply src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts:147]: Sure @princegupta1131 i'll update

File: src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts (hunk 2 above)
Comment (mithun30052001) [reply src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts:138]: As far as my knowledge we should subscribe the behavioursubject just like other sevices

**Pull request closed without merging**